source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -183,35 +186,6 @@ Create a symlink for the network overlays:
 
```

Comment:
Line 181: Can you please verify that this command works/has an effect? 
When I run this command, it fails.

Reply:
@DnPlas Can you please review this question too?

Reply:
What's the error?

Reply:
@mvincerx line numbers changed a bit, are you having trouble running 'sudo ln -s /usr/libexec/cni /opt/cni/bin'?

Reply:
Ah, I think I see it, it should be 'sudo ln -s /usr/libexec/cni /opt/bin/cni' (swap bin with cni). Could you please confirm @mythi ?